h2o-hadoop/h2o-cdh5.10/build.gradle

```diff
@@ -4,3 +4,10 @@ ext {
 }
 
 apply from: '../driverjar.gradle'
+
+dependencies {
+  compile (project(":h2o-jetty-8")) {
```

Comment:
Is this going to override the jetty version of CDH-5.10? I think we should exclude the transitive dependency on Jetty here or use other mechanism to let CDH define the correct dependency.